controlplane/kubeadm/controllers/upgrade.go

```diff
@@ -110,6 +111,20 @@ func (r *KubeadmControlPlaneReconciler) upgradeControlPlane(
 		return ctrl.Result{}, errors.Wrap(err, "failed to upgrade kubelet config map")
 	}
 
+	// this should be already handled by the defaulting webhook, but during rolling upgrade it is possible that
```

Comment:
I'm not sure if we still need this. 
This was for the CAPI upgrade when RolloutStrategy was introduced, the clusters created before that upgrade does not have the strategy, so we need to make sure after that upgrade the strategy should be set.
For all the clusters we have right now, they should already have this set, so probably don't need this change.